admin/spec/front-end/pupil-access-arrangements-selection.spec.js

```diff
@@ -85,17 +85,17 @@ function initElemets () {
 
   const $questionReaderReasons =
     `<div class="hide-checkbox-content">
-        <div class="form-group">${questionReaderReasons.map(function (qrr, i) {
-          return `<div class="multiple-choice">
-                <input id="questionReaderReason-${i}" type="radio" name="questionReaderReason"
-                    class="question-reader-reason" value="${qrr.code}">
-                    <label for="questionReaderReason-${i}">${qrr.title}</label>
-            </div>
-        ${qrr.code === 'OTH' ? $questionReaderOtherTextArea : ''}`
-        }
-    )}
+      <div class="form-group">${questionReaderReasons.map(function (qrr, i) {
+    return `<div class="multiple-choice">
+      <input id="questionReaderReason-${i}" type="radio" name="questionReaderReason"
+      class="question-reader-reason" value="${qrr.code}">
+        <label for="questionReaderReason-${i}">${qrr.title}</label>
       </div>
-    </div>`
```

Comment:
Not sure what's the change here even with the hide whitespace option it seems it's mostly indentation?

Reply:
This was a linting fix IIRC